README.md

```diff
@@ -27,14 +25,16 @@ Available make targets can be listed by running `make`.
 ### Running with docker
 
 1. Install [Docker](https://docs.docker.com/get-docker/)
-2. If you don't have Go installed, the makefile will automatically use Docker. If you do have go installed but still want to use docker, then set the `USE_GO_CONTAINERS` to `true`.
-
-Make targets will run with the *cfplatformeng/csb* docker image. Alternatively, a custom image can be specified by setting the `CSB` environment variable.
+2. Launch an interactive shell into some supported image containing all necessary tools. For example:
+   ```
+   # From the root of this repo run:
+   docker run -it --rm -v "${PWD}:/repo" --workdir "/repo" --entrypoint "/bin/bash" cflondonservices/csb-ci-main
```

Comment:
As we no longer update that image on Dockerhub, might it be better to point to another one? `golang` for example?

Reply:
Definitely, but some tests might not work since they use Docker to run:
https://github.com/cloudfoundry/csb-brokerpak-aws/blob/212c60f1378de4388026eab377ffb5ed3d0f9633/providers/terraform-provider-csbdynamodbns/resource_dynamodb_instance_test.go#L32

That's why I used `cflondonservices/csb-ci-main` as an example, but still stated that you need "some supported image containing all necessary tools".

Reply:
Discussed. This problem existed before and we are not making it worse.